app/client/components/creditTransfer/CreditTransferList.tsx

```diff
@@ -0,0 +1,297 @@
+import * as React from "react";
+
+import { Link } from "react-router-dom";
+
+import { Table, Tag, Pagination, Space } from "antd";
+
+import { ColumnsType } from "antd/es/table";
+
+import { ReduxState } from "../../reducers/rootReducer";
+import { connect } from "react-redux";
+import { formatMoney } from "../../utils";
+import DateTime from "../dateTime";
+
+interface StateProps {
+  users: any;
+  tokens: any;
+}
+
+export interface OnPaginateChange {
+  (page: number, pageSize: number | undefined): void;
+}
+
+export interface Pagination {
+  currentPage: number;
+  items: number;
+  onChange: OnPaginateChange;
+}
+
+interface stringIndexable {
+  [index: string]: any;
+}
+
+interface OuterProps extends stringIndexable {
+  params: string;
+  searchString: string;
+  creditTransfers: ReduxState["creditTransfers"];
+  users: ReduxState["users"];
+}
+
+export interface TransferAccount {
+  id: string;
+  created: string;
+  amount: string;
+  sender: string;
+  recipient: string;
+  transfer_type: string;
+  approval: string;
+  blockchain_approval: string;
+  transfer_status_colour: string;
+  blockchain_status_colour: string;
+}
+
+type Props = StateProps & OuterProps;
+
+const columns: ColumnsType<TransferAccount> = [
+  {
+    title: "ID",
+    key: "id",
+    ellipsis: true,
+    render: (text: any, record: any) => (
+      <Link
+        to={"/transfers/" + record.id}
+        style={{
+          textDecoration: "underline",
+          color: "#000000a6",
+          fontWeight: 400
+        }}
+      >
+        {record.id}
+      </Link>
+    )
+  },
+  {
+    title: "Created",
+    key: "created",
+    render: (text: any, record: any) => (
+      <DateTime created={record.created} useRelativeTime={false} />
+    )
+  },
+  {
+    title: "Amount",
+    key: "amount",
+    render: (text: any, record: any) => {
+      const money = formatMoney(
+        record.amount / 100,
+        undefined,
+        undefined,
+        undefined,
+        record.token_symbol
+      );
+      return <p style={{ margin: 0 }}>{money}</p>;
+    }
+  },
+  {
+    title: "Sender",
+    key: "sender_name",
+    ellipsis: true,
+    render: (text: any, record: any) => (
+      <Link
+        to={"/users/" + record.sender_user_id}
```

Comment:
nit: potential small change for usability. Have this go to `/accounts/` + transfer_account.id. As it breaks the breadcrumb flow
<img width="897" alt="Screen Shot 2021-07-07 at 8 17 10 am" src="https://user-images.githubusercontent.com/27039316/124675403-c9479000-defb-11eb-9180-fa1c7410560c.png">